float32 goUpDistance
---
int32 ExecStatus
